requirements.txt

```diff
@@ -1,16 +1,16 @@
 colorama==0.3.9
-urllib3[secure]==1.24.2
+urllib3[secure]>=1.26.5
 prettytable==0.7.2
```

Comment:
It is not linked with this PR :-)
But I often have issues with that prettytable requirement...
It is not in line with pyOCD: 'prettytable>=2.0,<3.0',
https://github.com/pyocd/pyOCD/blob/master/setup.py#L74

Maybe we can relax the version number here ?
Thx

Reply:
We need the higher version to get security fixes, so we can't relax. We can update pyOCD.

Reply:
pyOCD requests a higher version...

Reply:
I followed up on our CI failure with the newer prettytable. We had pinned mbed-os-tools's requirement on pretty table as documented in a previous commit:

If we want to update prettytable, we need to drop Python 2.7 and 3.5 support.
```
     Latest release of `PrettyTable` (2.0.0) removed support for Python 2.7
    and 3.5 and is causing CI failures. We are still supporting those
    versions in mbed-os-tools. Therefore pin `PrettyTable` to 1.0.1.
```

Reply:
If we drop Python 2.7 and 3.5 (which are deprecated), could we also add `cysecuretools`? This package is needed by CYTFM_064B0S2_4343W (and one or two other Cypress targets if I remember correctly). We didn't add it because it only supports Python 3.6 and above.

Reply:
We should be able to drop 2.7 and 3.5 (I thought 2.7 was already no supported).

Reply:
Dropping python2.7 will involve coordinated changes to many repos. I'll file an issue for doing this separately, including updating prettytable.